### PR TITLE
Random to crypto

### PR DESCRIPTION
**Pull request by mihmul83**
Replace Math.random() to cryptographically strong pseudo-random crypto.randomBytes
Replace Math.floor to bitwise or

### CODE REVIEW

**Comment by mihmul83** [comment]
I think that maybe to somehow soften the conditions for the Eslint rule "linebreak-style". Because I had problems with it on windows and in the end I made 5 commits for 4 linesof code, that makes me a little sad. Maybe we should add the file .eslintignore in .gitignore?

File: lib/sessions.js [hunk 1]
```diff
@@ -17,9 +19,10 @@ const sessions = new Map();
 
 const generateToken = () => {
   const base = ALPHA_DIGIT.length;
+  const bytes = crypto.randomBytes(base);
   let key = '';
   for (let i = 0; i < TOKEN_LENGTH; i++) {
-    const index = Math.floor(Math.random() * base);
+    const index = ((bytes[i] * base) / BYTE) | 0;
```
Comment (tshemsedinov) [inline lib/sessions.js:25]: I am ok with it, but `x | 0` truncates `x`, it's asm.js equivalent for `Math.truncate`. I think in this case it is not important. Also we have `~~x` and it's also suitable here but not readable.

**Comment by tshemsedinov** [comment]
@mihmul83 line breaks are not a problem, it is a standard in programming and all tools have workaround for windows, for example git have `autocrlf` configuration parameter, see: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration
Also all IDE supports different line ending, most of them have autodetection but try to configure it in IDE preferences.

**Comment by tshemsedinov** [comment]
I am going to land this PR

File: lib/sessions.js (hunk 1 above)
Comment (mihmul83) [reply lib/sessions.js:25]: I was guided by the fact that "bitwise or" seems to be the fastest way (~2% than Math(floor or trunc), by sometimes jsperf tests x | 0 fastest than Math.floor or Math.trunk at 30%)

Comment (tshemsedinov) [reply lib/sessions.js:25]: It's ok here but in this starter kit I am trying to write readable code, not performance-optimized

**Comment by tshemsedinov** [comment]
Landed in https://github.com/HowProgrammingWorks/NodejsStarterKit/commit/257cabd27e1fed613504e08e3cbf853a3a5a6f60

**Comment by tshemsedinov** [comment]
Squashed and landed, @mihmul83, thanks for your contribution!

**Pull request closed without merging**